code_challenges/code_wars/find_unique_number.py

```diff
@@ -0,0 +1,17 @@
+# https://www.codewars.com/kata/585d7d5adb20cf33cb000235/train/python
+
+
+def find_unique_number(numbers):
+    frequency_of_numbers = {}
+    for single_number in numbers:
```

Comment:
Food for thought: `for number in numbers:` would probably be a fine name here and prevent the wrapping on line 7-8